animata/card/card-comment.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { CardComment } from "@/animata/card/card-comment";
+import { Meta, StoryObj } from "@storybook/react";
+
+const meta = {
+  title: "Card/Card Comment",
+  component: CardComment,
+  parameters: {
+    layout: "centered",
+  },
+  tags: ["autodocs"],
+  argTypes: {},
+} satisfies Meta<typeof CardComment>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Primary: Story = {
+  args: {
+    commenter: "Anna ",
+    replier: "Sumana",
+  },
+};
+export const Secondary: Story = {
+  args: {
+    commenter: "Radhey",
+    replier: "Krishna",
+  },
+};
```

Comment:
you can get rid of the second story as it doesn't add much value @Abishkardhenga

Reply:
ok